schema/schema.js

```diff
@@ -0,0 +1,184 @@
+const graphql = require("graphql");
+// const _ = require("lodash");
+const User = require("../models/user.js");
+const Location = require("../models/location.js");
+const {
+  getGraphQLQueryArgs,
+  getMongoDbQueryResolver,
+  getGraphQLUpdateArgs,
+  getMongoDbUpdateResolver,
+  getGraphQLInsertType,
+  getGraphQLFilterType,
+  getMongoDbFilter
+} = require("graphql-to-mongodb");
+const {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLSchema,
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLFloat
+} = graphql;
+
+const LocationType = new GraphQLObjectType({
+  name: "Location",
+  fields: () => ({
+    ID: { type: GraphQLID },
+    DISTRICT: { type: new GraphQLNonNull(GraphQLString) },
+    CountyNum: { type: new GraphQLNonNull(GraphQLInt) },
+    COUNTY: { type: new GraphQLNonNull(GraphQLString) },
+    NameMobileWeb: { type: new GraphQLNonNull(GraphQLString) },
+    LocationMobileWeb: { type: new GraphQLNonNull(GraphQLString) },
+    DescriptionMobileWeb: { type: new GraphQLNonNull(GraphQLString) },
+    PHONE_NMBR: { type: new GraphQLNonNull(GraphQLString) },
+    PARKING: { type: new GraphQLNonNull(GraphQLString) },
+    DSABLDACSS: { type: new GraphQLNonNull(GraphQLString) },
+    RESTROOMS: { type: new GraphQLNonNull(GraphQLInt) },
+    VISTOR_CTR: { type: new GraphQLNonNull(GraphQLString) },
+    DOG_FRIENDLY: { type: new GraphQLNonNull(GraphQLString) },
+    EZ4STROLLERS: { type: new GraphQLNonNull(GraphQLString) },
+    PCNC_AREA: { type: new GraphQLNonNull(GraphQLString) },
+    CAMPGROUND: { type: new GraphQLNonNull(GraphQLString) },
+    SNDY_BEACH: { type: new GraphQLNonNull(GraphQLString) },
+    DUNES: { type: new GraphQLNonNull(GraphQLString) },
+    RKY_SHORE: { type: new GraphQLNonNull(GraphQLInt) },
```

Comment:
Having the same issue as above with the type not allowing me to query this field.